btrdb/stream.py

```diff
@@ -1145,6 +1151,26 @@ def windows(self, width, depth):
         specified. This is much faster to execute on the database side.
 
         """
+        start, end = None, None
+        if len(args) < 2 and len(kwargs) < 2:
+            raise UserWarning("Require 'width' and 'depth'")
+        elif len(args) == 2 and len(kwargs) == 0:
+            width, depth = args
+        elif len(args) == 0 and len(kwargs) == 2:
```

Comment:
I think that, if the user gave start & end in the kwargs but noth width & depth, this could lead to unclear errors.